src/DashboardService.Data.Provider.MsSql.Ef/DashboardServiceDbContext.cs

```diff
@@ -0,0 +1,51 @@
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.Kernel.EFSupport.Provider;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
```

Comment:
I am sure it's unused. Use Ctrl+G to get rid of unused 'usings'

Reply:
Assembly class in using System.Reflection;
modelBuilder.ApplyConfigurationsFromAssembly(**Assembly**.Load("LT.DigitalOffice.DashboardService.Models.Db"));

Reply:
Did you add it yourself? Not sure if it's required since we don't implement it and write migrations manually

Reply:
i use System.Reflection for import Assembly class.
![image](https://github.com/LT-Students/DashboardService/assets/42372955/c3eb8c9c-0895-44bd-beb9-3a3140ae3443)

or i didnt understand your comment)

Reply:
nevermind